source/shared/net/desktop-wallet/change-baker-stake.rst

```diff
@@ -18,7 +18,7 @@ Change the staked amount
 
 #. Go to **Multi Signature Transactions**, select **Make new proposal**, and then select **Update baker stake**.
 
-#. Select the identity that’s associated with the account whose baker stake you want to change, and then select **Continue**.
+.. #. Select the identity that’s associated with the account whose baker stake you want to change, and then select **Continue**.
```

Comment:
It looks like this now. I don't suppose it is supposed to say 1. twice, and have the whitespace between the two?

![image](https://user-images.githubusercontent.com/76045015/123596424-77638380-d7f2-11eb-933c-97e9de4ebbf6.png)